src/widget/wtracktableview.cpp

```diff
@@ -616,25 +622,72 @@ void WTrackTableView::slotMouseDoubleClicked(const QModelIndex &index) {
 
 void WTrackTableView::loadSelectionToGroup(QString group, bool play) {
     QModelIndexList indices = selectionModel()->selectedRows();
```

Comment:
Why is this function being modified at all in this PR? This will cause merge conflicts with #2612.

Reply:
Because it used to do an assignent inside an if condition expression.

Reply:
Oh I understand. You copied and pasted from this function then went back and fixed both your new functions and this old function. Please reset your branch to 0657f14f969a6b708f6af4d24319efffe2d16db7, only commit the changes to your new functions, and force push to avoid merge conflicts with #2612. The assignment in the `if` statement is fixed in a different way in #2612 (in WTrackMenu).

Reply:
#2612 has been merged now, so the merge conflicts must be fixed before this can be merged.